base/applications/rapps/geninst.cpp

```diff
@@ -178,10 +178,7 @@ ErrorBox(UINT Error = GetLastError())
 {
     if (!Error)
         Error = ERROR_INTERNAL_ERROR;
-    WCHAR buf[400];
-    UINT fmf = FORMAT_MESSAGE_IGNORE_INSERTS | FORMAT_MESSAGE_FROM_SYSTEM;
-    FormatMessageW(fmf, NULL, Error, 0, buf, _countof(buf), NULL);
-    MessageBoxW(g_pInfo->GetGuiOwner(), buf, 0, MB_OK | MB_ICONSTOP);
+    ErrorBox(g_pInfo->GetGuiOwner(), Error);
```

Comment:
```suggestion
    Error = ErrorBox(g_pInfo->GetGuiOwner(), Error);
```
take the opportunity that the new ErrorBox already normalizes the error to `ERROR_INTERNAL_ERROR` if needed, and returns that normalized error.